cli/flox/src/commands/environment.rs

```diff
@@ -436,7 +436,7 @@ impl Activate {
             environment.activation_path(&flox)?
         } else {
             Dialog {
-                message: &format!("Building environment '{prompt_name}'..."),
+                message: &format!("Getting ready to use environment '{prompt_name}'..."),
```

Comment:
suggestion: ticket asks for name at location format, not just name

Reply:
```suggestion
                message: &format!("Getting ready to use environment '{now_active}'..."),
```